src/status_im2/contexts/wallet/add_address_to_watch/view.cljs

```diff
@@ -66,6 +69,35 @@
        :icon-only?      true}
       :i/scan]]))
 
+(defn activity-indicator
+  []
+  (let [activity-state                                  (rf/sub [:wallet/watch-address-activity-state])
+        {:keys [accessibility-label icon type message]} (case activity-state
```

Comment:
maybe we can put `case` on the next line and this will format much nicer? 
```
:accessibility-label
:account-has-activity
``` 
formatting is sitting a bit strangely at the moment

Reply:
Updated 👍